signature/src/error.rs

```diff
@@ -1,7 +1,7 @@
 //! Signature error copied directly from
 //! [RustCrypto's opaque signature error](https://github.com/RustCrypto/traits/tree/master/signature)
 
-#[cfg(all(feature = "std", feature = "alloc"))]
+#[cfg(all(feature = "alloc"))]
```

Comment:
nit: for all of these, we don't need `all` anymore
```suggestion
#[cfg(feature = "alloc")]
```